app/src/components/page/Sidebar/index.tsx

```diff
@@ -136,7 +139,7 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
           isAuthorized(roles as string[], activeRoles.INVENTORY.split(',')) && (
             <Menu.SubMenu
               key="inventory"
-              icon={<ArchiveOutline className="sidebar-icons" />}
+              icon={<InboxOutlined className="sidebar-icons" />}
```

Comment:
@ashfahan please confirm that these icons are the ones that are correct, the original issue is here: https://github.com/OpenSRP/web/issues/251

Reply:
Regarding the Missions (Map pin) icon the one that i used it the same like we are using in opensrp right now (which is slightly different than the one provided on the original issue) and for the Inventory icon the one that i have replaced with it slightly different then the one we have right now but thats the closest one i could find in the ant design icons

Reply:
@ashfahan you can check out this lib `@2fd/ant-design-icons`, I found it has a more extensive icons offering than ant-design/icons

Reply:
wasn't the goal of this pr to only have ant icons only and remove other dependencies ?